### PR TITLE
Add the documentation website link to README

### PR DESCRIPTION
**Pull request by tirthasheshpatel**
Add the documentation website link to `README.md`

@tqchen @rgommers

### CODE REVIEW

**Review by tqchen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://github.com/dmlc/dlpack/actions/workflows/main.yaml/badge.svg?branch=main)](https://github.com/dmlc/dlpack/actions/workflows/main.yaml)
 
+Documentation: [https://dmlc.github.io/dlpack/latest](https://dmlc.github.io/dlpack/latest)
```
Comment (edgarriba) [inline README.md:5]: worth adding that link in the right side of the gh repo under project description @tqchen

Comment (tirthasheshpatel) [reply README.md:5]: I agree, that would be helpful!

Comment (tqchen) [reply README.md:5]: updated!

Comment (tirthasheshpatel) [reply README.md:5]: Thanks @tqchen!

**Pull request merged**